src/app.scss

```diff
@@ -0,0 +1,260 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+span,
+label,
+input,
+button,
+a {
+    font-family: 'Ropa Sans', sans-serif;
+}
+  .categoryContainer{
+    display:flex;
+    width:100%;
+    flex-direction: column;
+    padding-right: 5px;
+    padding-left: 5px;
+    margin-right: auto;
+    margin-left: auto;
+    .categoryCard{
+      display:grid;
+      grid-template-columns:minmax(33.33%,33.33%) minmax(66.66%,66.66%);
+      padding: 2rem 0;
+      margin-top: 2rem;
+      margin-bottom: 2rem;
+      box-shadow: 0 13px 25px -23px #a5a5a5ad;
+    }
+    .productCard{
```

Comment:
css should be break out in different component, every component should have it's own css